tests/cypress/support/commands.js

```diff
@@ -41,6 +41,40 @@ Cypress.Commands.add('userRegistration', (firstName, lastName, userName, emailAd
     }
 });
 
+Cypress.Commands.add('deletingRegisteredUsers', () => {
+    cy.request({
+        method: 'POST',
+        url: '/api/v1/auth/login',
+        body: {
+            username: Cypress.env('user'),
+            email: Cypress.env('email'),
+            password: Cypress.env('password'),
+        },
+    }).then(async (responce) => {
+        const authKey = await responce['body']['key'];
```

Comment:
Is this operation really ``async``? Should we use ``await``?

Reply:
Yes, this is not necessary here.